pinot-integration-tests/src/test/resources/tpch/22.sql

```diff
@@ -21,7 +21,7 @@ where
       )
       and ps_availqty > (
         select
-          0.5 * sum(l_quantity)
+          CAST(0.5 * SUM(l_quantity) AS DECIMAL(19, 1))
```

Comment:
the problem with this is b/c the 0.5 has DECIMAL value and it is a Literal 
this will cause Calcite to explicitly hoist the SUM results by a specific precision and scale. 

no one will actually use this in practices, and i suggest we disable this test until further fix can be put in place?

Reply:
The issue here is for join schema mismatch from both side, one is `ps_availqty` the type is `decimal(19, 1)`, the other side is `0.5 * SUM(l_quantity)`, which is `decimal(21,1)`.